cypress/fixtures/itemLikes.js

```diff
@@ -0,0 +1,15 @@
+import { PUBLISHED_ITEMS } from "./items";
+import { MEMBERS } from "./members";
+
+export const ITEM_LIKES = [
+  {
+    itemId: PUBLISHED_ITEMS[0].id,
+    memberId: MEMBERS.ANNA.id,
+  },
+  {
+    itemId: PUBLISHED_ITEMS[1].id,
+    memberId: MEMBERS.ANNA.id,
+  }
+]
+
+export const LIKED_ITEM_NUMBER = 1;
```

Comment:
Can't we dynamically compute this number? It's obviously related to the number of liked items you return.